lib/__tests__/is-visible.ts

```diff
@@ -32,21 +32,6 @@ describe('assert.dom(...).isVisible()', () => {
         },
       ]);
     });
-
-    test('fails if element is missing', () => {
```

Comment:
[no-identical-title](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-identical-title.md), was a duplicate test case.

Reply:
hmm, I think the original intention here was that the first test was the happy path with `assert.dom('h1').isVisible();` 😅

Reply:
I see. I just checked and it looks like we can't test the happy path here due to:

```
 * JSDom based tests aren't able to discern visibility as we define it. Specifically,
 * the JSDom tests don't do layouting, therefore calculating `offsetWdith` or `offsetHeight`
 * won't work. As a result, we need to use Ember's test infrastructure to correctly assess
 * visibility, as those tests run in a browser environment.
 *
 * Additional tests cases of isVisible can be found in tests/acceptance/qunit-dom-test.js
```

So removing the test case should be fine.

Reply:
makes sense, yeah